multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -277,6 +286,17 @@ def create_fusion_data_processors(
             )
             if data_types is not None and IMAGE in data_types:
                 data_types.remove(IMAGE)
+        elif per_name == OVD:
+            # create a multimodal processor for OVD.
+            data_processors[OVD].append(
+                create_data_processor(
+                    data_type=OVD,
+                    config=config,
+                    model=per_model,
+                )
+            )
+            if data_types is not None and IMAGE in data_types:
+                data_types.remove(IMAGE)
```

Comment:
`OVD` is not a data type. If there are no correlations for processing image and text, we can create an image processor and a text processor for the ovd model.

Reply:
There will be  correlations for processing image and bounding boxes if we support training. Will refactor out text processor in next PR. And will keep OVD data type until we combine this and traditional detection (mmdet) to both use ROIS data type.